### ISSUE TITLE
Add support to plugins

### ISSUE THREAD

**Issue by duviful** [jvoss/ansible-role-netbox#27]
Hello dear sir.
it would be nice to add custom plugins to `local_requirements.txt` and install them automatically.

a task in the file `configure.yml` similar to this one can do the job:

```yaml
- name: configure | add the custom plugins local_requirements.txt
  ansible.builtin.lineinfile:
    path: "{{ netbox_current_path }}/local_requirements.txt"
    line: "{{ item.name }}"
    regexp: "^{{ item.name }}"
    owner: "{{ netbox_user }}"
    mode: '0644'
    create: yes
  with_items: "{{ netbox_plugins }}"
```

I'm using the already present variable `netbox_plugins` to generate the list of names to be added to the `local_requirements.txt`

I might open a PR but I suspect you are already working on something like this.
let me know.

Best

**Comment by duviful** [comment]
Some plugins have a `pip` package name that differs from the `PLUGIN_CONFIG` name.
Probably would be better to use a separate variable `netbox_local_requirements_plugin` for the `local_requirement.txt` population.

**Comment by jvoss** [comment]
Hi! I was working on plugins at one point and got side tracked. I would certainly welcome a PR to add this functionality to the role.

You bring up a good point on the package name differing from the plugin name. Additionally, we would also have to ensure any required plugin steps are followed per the [docs](https://demo.netbox.dev/static/docs/plugins/), ie. database migration, etc. 

Also need to ensure this does not conflict with itself during installation, upgrades, config changes, etc.

**Comment by jvoss** [comment]
Found some time to work on plugins again. The linked PR uses a similar method to what you were proposing. I cross-referenced the `upgrade.sh` script with how NetBox recommends installing packages. Fortunately the script completes all the same tasks for us, so I used changes here to trigger that script to run again.

The PR should successfully install and configure as well as allow removal of plugins as required. Let me know if you experience any issues with this.
